### PR TITLE
Fixed add w2ui-input class to w2field

### PR DESCRIPTION
**Pull request by hawkxu**
Additional: add outline-style, outline-width to w2ui-input:focus

### CODE REVIEW

File: src/less/src/common.less [hunk 1]
```diff
@@ -79,6 +79,8 @@ input:not([type=button]):not([type=submit]).w2ui-input, textarea.w2ui-input {
     background-color: @input-background-color;
     line-height: normal;
     &:focus {
+        outline-style: solid;
+        outline-width: 1px;
```
Comment (vitmalina) [inline src/less/src/common.less]: Why don't you like regular focus style. I think that shadow kind of style looks good. You are welcome to write a custom css style that you can load on top of regular.

File: src/w2form.js [hunk 2]
```diff
@@ -1032,8 +1032,6 @@
                     case 'email':
                     case 'pass':
                     case 'password':
-                        field.el.value = value;
-                        break;
```
Comment (vitmalina) [inline src/w2form.js]: Above fields do not have options, therefore there is no point for it to have options object.

Comment (hawkxu) [inline src/w2form.js]: I add the class w2ui-input through the w2field init function, the line 1036 cause the above type field bypass w2field.init function.

File: src/less/src/common.less (hunk 1 above)
Comment (hawkxu) [inline src/less/src/common.less]: sorry, I used firefox 41.0.2, without the two attributes, the focus field no any outline border appeared, I did not test it with other browser.

File: src/w2form.js (hunk 2 above)
Comment (hawkxu) [inline src/w2form.js]: Please take a look at this: http://plnkr.co/edit/tpl:heAMIR7Yae9GJCXW70nP?p=preview

File: src/less/src/common.less (hunk 1 above)
Comment (vitmalina) [inline src/less/src/common.less]: Chrome. 

I looked up and it seems that in FF it has nice focus border, not 1px though. Plus this code will break nice border in chrome, which is with a blur.

File: src/w2form.js (hunk 2 above)
Comment (vitmalina) [inline src/w2form.js]: I think it is a wrong place to do that. I have added it in refresh function

**Comment by vitmalina** [comment]
I applied some changes

**Pull request closed without merging**